### ISSUE TITLE
Included trivial-garbage causes trouble – can be removed

### ISSUE THREAD

**Issue by metayan** [lispbuilder/lispbuilder#65]
The included `trivial-garbage` is old (version 0.17 dated 2008-10-27) and causes the following trouble to for example `clim-listener` when `lispbuilder` is present in `quicklisp/local-projects`:
```
debugger invoked on a SB-EXT:UNKNOWN-KEYWORD-ARGUMENT in thread
#<THREAD "main thread" RUNNING {10005505B3}>:
  Unknown &KEY argument: :WEAKNESS-MATTERS
```
`lispbuilder` compiles and works fine even when `lispbuilder-sdl/trivial-garbage.asd` and the whole `lispbuilder-sdl/trivial-garbage` directory is removed.

**Comment by elliottslaughter** [comment]
Yes, this seems reasonable to me. I'd be happy to accept pull requests for this.

**Comment by elliottslaughter** [comment]
Fixed by 620f51a0e39391ae7a05bc90a3b11f80579654fa
